src/services/azblob/backend.rs

```diff
@@ -574,6 +574,31 @@ impl Accessor for AzblobBackend {
 
         Ok((RpScan::default(), op))
     }
+
+    async fn batch(&self, args: OpBatch) -> Result<RpBatch> {
+        let ops = args.into_operation();
+        match ops {
+            BatchOperations::Delete(ops) => {
+                // even the Azure CLI implements batch delete by deleting files
```

Comment:
Hello, this prediction is incorrect. OpenDAL's batch operation is specifically designed for performing tasks in batches. Therefore, implementing a batch delete function would be redundant since users can already perform individual deletions.

I prefer to see a native batch support via: [Blob Batch](https://learn.microsoft.com/en-us/rest/api/storageservices/blob-batch)

Reply:
Thanks for your reminder, working on it.

Reply:
This requires implementing `OData` batch API, but I don't currently see any rust http library supports currently.

related issue:
- https://github.com/seanmonstar/reqwest/issues/1241 : no responce

Reply:
I am not looking for anything fancy. Please provide the content in plain text format without requiring extensive support. Write a simple txt generate which will output content like :

```txt
--batch_357de4f7-6d0b-4e02-8cd2-6361411a9525 
Content-Type: application/http 
Content-Transfer-Encoding: binary 
Content-ID: 0 

DELETE /container0/blob0 HTTP/1.1 
x-ms-date: Thu, 14 Jun 2018 16:46:54 GMT 
Authorization: SharedKey account:G4jjBXA7LI/RnWKIOQ8i9xH4p76pAQ+4Fs4R1VxasaE= 
Content-Length: 0
```